services/web/server/src/simcore_service_webserver/studies_dispatcher/_studies_access.py

```diff
@@ -345,9 +344,21 @@ async def get_redirection_to_study_page(request: web.Request) -> web.Response:
         # NOTE: covers valid cookie with unauthorized user (e.g. expired guest/banned)
         user = await get_authorized_user(request)
 
+    # This was added so it fails right away if study doesn't exist.
```

Comment:
@matusdrobuliak66, I've added this (lines 347-358) to make it fail right away with a 404 when a study doesn't exist. This fixed the test: `services/web/server/tests/unit/with_dbs/01/studies_dispatcher/test_studies_dispatcher_studies_access.py`, `test_access_to_invalid_study`. 
Let me know if see something dangerous with with this addition.